src/pages/user_profile/UserProfile.js

```diff
@@ -38,6 +59,7 @@ export default function UserProfile(props) {
           email: user.email,
           id: user.userId
         });
+        setName(user.displayName)
```

Comment:
nit: missing semicolon
```suggestion
        setName(user.displayName);
```